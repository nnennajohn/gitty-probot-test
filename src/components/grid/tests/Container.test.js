// @flow
import React from 'react';
import renderer from 'react-test-renderer';

import { Container } from '../index';

test('Styles matches Container', () => {
  const tree = renderer.create(<Container />).toJSON();
  expect(tree).toMatchSnapshot();
  // expect(tree).toHaveStyleRule('margin-right', 'auto');
  // expect(tree).toHaveStyleRule('margin-left', 'auto');
  // expect(tree).toHaveStyleRule('box-sizing', 'border-box');
  // expect(tree).toHaveStyleRule('width', '100%');
});
