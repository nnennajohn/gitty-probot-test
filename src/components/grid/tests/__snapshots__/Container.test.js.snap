// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches Container 1`] = `
.c0 {
  width: 100%;
  margin-right: auto;
  margin-left: auto;
  box-sizing: border-box;
}

@media only screen and (min-width:40em) {
  .c0 {
    max-width: 46rem;
  }
}

@media only screen and (min-width:64em) {
  .c0 {
    max-width: 61rem;
  }
}

@media only screen and (min-width:75em) {
  .c0 {
    max-width: 76rem;
  }
}

<div
  className="c0"
/>
`;
